<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mesh setup visibility</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>270</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>360</width>
     <height>500</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>40</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>6</y>
       <width>300</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>User interface visibility settings</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton_confirm">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>460</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>52</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Model</string>
     </attribute>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>240</y>
        <width>240</width>
        <height>120</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>120</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="checkBox_elements_selector">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>50</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Elements</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_nodes_selector">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>14</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> Nodes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_lines_selector">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>86</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Lines</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>40</y>
        <width>240</width>
        <height>160</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>160</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="checkBox_structural_symbols_viewer">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>122</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Structural symbols</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_elements_viewer">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>50</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Elements</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_nodes_viewer">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>14</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nodes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_acoustic_symbols_viewer">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>86</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Acoustic symbols</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>210</y>
        <width>240</width>
        <height>32</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Enable/Disable selection</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>10</y>
        <width>240</width>
        <height>32</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Show/Hide</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Advanced</string>
     </attribute>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>240</y>
        <width>240</width>
        <height>120</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>120</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="checkBox_MOPT_logo">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>50</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MOPT logo</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_OpenPulse_logo">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>14</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>OpenPulse logo</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_reference_scale">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>86</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reference scale</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>40</y>
        <width>240</width>
        <height>160</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>160</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QRadioButton" name="radioButton_light_gray_color">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>86</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Light gray</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButton_black_color">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>14</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Black</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButton_white_color">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>122</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>White</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButton_dark_gray_color">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>50</y>
         <width>170</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Dark gray</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>210</y>
        <width>240</width>
        <height>32</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Show/Hide</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>10</y>
        <width>240</width>
        <height>32</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Background color</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
